modules/javafx.graphics/src/main/native-glass/mac/a11y/AccessibleBase.m

```diff
@@ -50,6 +50,7 @@ + (void) initializeRolesMap {
     [rolesMap setObject:@"JFXRadiobuttonAccessibility" forKey:@"PAGE_ITEM"];
     [rolesMap setObject:@"JFXCheckboxAccessibility" forKey:@"CHECK_BOX"];
     [rolesMap setObject:@"JFXCheckboxAccessibility" forKey:@"TOGGLE_BUTTON"];
+    [rolesMap setObject:@"JFXStaticTextAccessibility" forKey:@"TEXT"];
```

Comment:
are these keys mapped to AccessibleRole.java enum?  if so,
1. should we add a comment stating that (and possibly add a comment in AccessibleRole as well?
2. are we going to keep adding roles to this block as we go along?

Reply:
1. Yes, these are linked to the JFX accessible roles defined in that enum. I can add comment here - not sure about the AccessibleRole, it is a definition that might be used for more than one platform so linking platform specific implementation from there sound incorrect;
2. Yes, as we are converting more and more functionality to the new API we are going to add roles implementations to this block until we have no roles not covered with the new API.

Reply:
thank you for clarifying.  fully agree with #1, referencing from platform specific to the common should be sufficient.